### DISCUSSION TITLE
Jerrycans to be added to the build menu / function

### DISCUSSION THREAD

**Discussion by raypot28** [KillahPotatoes/KP-Liberation#889] [General]
Hi All,

Any options to have jerrycans added to the build function/menu. I know you can spawn them via Zeus but on server restart they
will despawn....

Thank you

**Comment by Beeperz** [comment]
Add it's classname to your fraction preset, in support section I think:
https://github.com/KillahPotatoes/KP-Liberation/blob/master/Missionframework/presets/blufor/custom.sqf#L306
